### ISSUE TITLE
Wrong reporting of TestCaseName in the perf results database

### ISSUE THREAD

**Issue by ader1990** [LIS/LISAv2#515]
When uploading the performance results in the database, the TestCaseName is not properly inserted in the database. The TestCaseName value gets the testTag value from the secrets, which is a direct violation of the initial intent of the database creator.

Issues:
https://github.com/LIS/LISAv2/blob/master/Testscripts/Windows/PERF-NETWORK-THROUGHPUT-MULTICONNECTION-NTTTCP.ps1#L210
https://github.com/LIS/LISAv2/blob/master/Testscripts/Windows/PERF-STORAGE-MULTIDISK-RAID0-FIO.ps1#L126
https://github.com/LIS/LISAv2/blob/master/Testscripts/Windows/PERF-NETWORK-TCP-SINGLE-CONNECTION-THROUGHPUT.ps1#L96

@LiliDeng @lizzha @juhlee-microsoft Do you know the reason behind this workflow?

**Comment by LiliDeng** [comment]
@ader1990 It is assigned in the PS1 code
LISAv2\Testscripts\Windows\PERF-NETWORK-THROUGHPUT-MULTICONNECTION-NTTTCP.ps1 (1 hit)
	Line 210:         $TestCaseName = $xmlConfig.config.$TestPlatform.database.testTag

**Comment by lizzha** [comment]
@iamshital  could you please share the initial design of why the TestCaseName in database table comes from testTag value?

**Comment by iamshital** [comment]
@lizzha , We made it to support better powerbi filtering.

for example: We are running a full test pass against, LIS 4.2.8, so we used to tag all the functional / network / performance tests with tag - 'LIS-4.2.8'. And with one powerbi filter, we could see all results on one page. 
Earlier, there was no common column was present, that we could control externally to tag specific test case and later filter by it's execution tag.
So, we modified the test name with test tag. (If test tag was empty, it would take original test name).

By this way , we used to tag all the perf tests with single execution tag and which was very helpful for filtering later.

Now, I still want to have the functionality to tag our results with certain string. (if user wants). This can help us when creating custom powerbi reports / result excel sheets directly from DB. But it's not super urgent.
